install.sh

```diff
@@ -270,5 +284,8 @@ add_tags_to_asg_resources
 # Resume bamboo server if the credential is provided
 resume_bamboo_server
 
+# Set the correct Synchrony URL
+set_synchrony_url
```

Comment:
The output of this function if Confluence was installed and domain is not provided:

```
2022-02-15 12:26:28 [INFO] Configuring the Synchrony service.
2022-02-15 12:26:48 [INFO] Synchrony URL is set to 'http://ac8230a53e6ad4bacb27da6028483ec5-1351588498.ap-southeast-1.elb.amazonaws.com/synchrony'.
```